### PR TITLE
Fix Stanford: Programming Paradigms playlist broken link

### PR DESCRIPTION
**Pull request by gjzim**
The current Stanford: Programming Paradigms youtube playlist's [link](https://www.youtube.com/view_play_list?p=9D558D49CA734A02) is broken. This pull request updates it with a working [one](https://www.youtube.com/playlist?list=PL9D558D49CA734A02).

### CODE REVIEW

**Comment by jwasham** [comment]
Thank you!

File: README.md [hunk 1]
```diff
@@ -1965,7 +1965,7 @@ Sit back and enjoy.
 
 - [MIT 6.858 Computer Systems Security, Fall 2014](https://www.youtube.com/watch?v=GqmQg-cszw4&index=1&list=PLUl4u3cNGP62K2DjQLRxDNRi0z2IRWnNh)
 
-- [Stanford: Programming Paradigms (27 videos)](https://www.youtube.com/view_play_list?p=9D558D49CA734A02)
+- [Stanford: Programming Paradigms (27 videos)](https://www.youtube.com/playlist?list=PL9D558D49CA734A02)
 
 - [Introduction to Cryptography by Christof Paar](https://www.youtube.com/playlist?list=PL6N5qY2nvvJE8X75VkXglSrVhLv1tVcfy)
     - [Course Website along with Slides and Problem Sets](http://www.crypto-textbook.com/)
```
Comment (suiap) [inline README.md:1971]: gaiz

File: README.md [hunk 2]
```diff
@@ -1965,7 +1965,7 @@ Sit back and enjoy.
 
 - [MIT 6.858 Computer Systems Security, Fall 2014](https://www.youtube.com/watch?v=GqmQg-cszw4&index=1&list=PLUl4u3cNGP62K2DjQLRxDNRi0z2IRWnNh)
 
-- [Stanford: Programming Paradigms (27 videos)](https://www.youtube.com/view_play_list?p=9D558D49CA734A02)
+- [Stanford: Programming Paradigms (27 videos)](https://www.youtube.com/playlist?list=PL9D558D49CA734A02)
```
Comment (suiap) [inline README.md:1968]: ffff

File: translations/README-cn.md [hunk 3]
```diff
@@ -1828,7 +1828,7 @@
 
 - [MIT 6.858：计算机系统安全, 2014 年秋季](https://www.youtube.com/watch?v=GqmQg-cszw4&index=1&list=PLUl4u3cNGP62K2DjQLRxDNRi0z2IRWnNh)
 
-- [斯坦福: 编程范例 (27 个视频)](https://www.youtube.com/view_play_list?p=9D558D49CA734A02)
+- [斯坦福: 编程范例 (27 个视频)](https://www.youtube.com/playlist?list=PL9D558D49CA734A02)
```
Comment (suiap) [inline translations/README-cn.md:1831]: sss

**Review by suiap — COMMENTED** [review]
ssnj

**Pull request merged**